splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -279,6 +350,18 @@ def _get_and_check_global_config_path(source: str, config_path: Optional[str]) -
     return ""
 
 
+def _delete_pyc_files(lookup_path: str) -> None:
```

Comment:
nit : Instead of deleting each individual `.pyc` files, wouldn’t it be better to delete the whole `__pycache__` directory.